test/integration/patients/patient/data-and-events.js

```diff
@@ -208,9 +211,22 @@ context('patient data and events page', function() {
       .find('tr')
       .should('have.lengthOf', 3);
 
-    // TODO: Uncomment when patient flow app is implemented
-    // cy
-    //   .url()
-    //   .should('contain', 'flows/2');
```

Comment:
thoughts on the best way to handle needing to click through to new links on multiple items? A new specify block just for that?

Reply:
I think we can use `cy.go('back')` in some cases

Reply:
I tried that initially and it looked like it wasn't going to work...lemme play around with it more